fast64_internal/sm64/sm64_geolayout_bone.py

```diff
@@ -427,9 +427,14 @@ def draw(self, context):
 
 def getSwitchOptionBone(switchArmature):
     optionBones = []
-    for poseBone in switchArmature.pose.bones:
-        if poseBone.bone_group is not None and poseBone.bone_group.name == "SwitchOption":
-            optionBones.append(poseBone.name)
+    if bpy.app.version >= (4, 0, 0):
+        for bone in switchArmature.data.bones:
+            if bone.name in switchArmature.data.collections["SwitchOption"].bones:
```

Comment:
I'm discovering the new API, I think this can be simplified to
```suggestion
            if "SwitchOption" in bone.collections:
```
since it seems `bone.collections` is a bpy_collection with the collection names as keys